launchable/commands/subset.py

```diff
@@ -116,6 +116,11 @@ def separator(self, s: str):
             self._separator = s
 
         def test_path(self, path: TestPathLike):
+            if isinstance(path, str) and any(s in path for s in ('/**', '**/', '*.')):
```

Comment:
Do we need to consider `*`, `/*`case?

Reply:
if add `*`, `test/test_h*ge.rb` match this if statement and happen infinity loop error.
so I'll add only `/*`.

(Ofcource, I know `test/test_h*ge.rb`is rare case)

Reply:
@ninjinkun I added https://github.com/launchableinc/cli/pull/188/commits/2e2b75762e12131e9dcf468d09d58fbcb7391fb9

Reply:
Help me understand why this is not `s in path for s in ('*', '?')` and instead there are restrictions about where the wildcard appears. This feels like a rough edge waiting to bite somebody. I can totally imagine somebody doing something like `foo/*/*_test.py` somewhere.

Why would this result in an infinite loop? And even if it does, shouldn't we resolve that by not doing recursion?

Reply:
thank you. your point is right.

before I added this change (https:/github.comlaunchableincclipull188commits521288834e1f0c26ee6fdd512b6fcd58027704b6), an infinite loop error had happened.
but now, we checked the path is a directory or not, so it doesn't happen.

I'll fix it.